Server/src/models/entities/Material.ts

```diff
@@ -54,3 +54,11 @@ export const selectMaterialQuery = (manager: EntityManager, dataset: number) =>
         .innerJoin(Composition, 'composition', 'material.compositionId = composition.id')
         .where('dataset.id = :datasetId', { datasetId: dataset })
         .getRawMany();
+
+export const selectAllMaterialsQuery = (manager: EntityManager) =>

```

Comment:
Can we move all this into a model?

Reply:
Done. Earlier it was somewhat decided for consistency to put any queries, complete or partial, into their respective Entity files encase others needed to reference said queries. But, given that these particular queries will only be used for this service, in hindsight that logic does not apply in this case.